guide/ops/starting-stopping-monitoring.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Starting, Stopping and Monitoring
+layout: website-normal
+---
+
+**NOTE:** This document is for information on starting an Apache Brooklyn
+Server.  For information on using the Brooklyn Client CLI to access an already
+running Brooklyn Server, refer to [Client CLI Reference](cli/index.html).
+
+
+## Starting
+
+To launch Brooklyn, from the directory where Brooklyn is unpacked, run:
+
+{% highlight bash %}
+% bin/brooklyn launch > /dev/null 2>&1 & disown
```

Comment:
Looking more closely, I see that the server-cli-reference doesn't have a good "stop" section. Where do you think is most appropriate to put your new section on stopping? (bearing in mind we'll need two variants: one for this approach, and one for rpm/deb packages).